docs/pages/guides/ci-cd-pipelines.mdx

```diff
@@ -162,4 +164,370 @@ RUN chmod +x /bin/loft
 
 ### Login with Access Keys
 
-<LoginAccessKeysFragment/>
\ No newline at end of file
+<LoginAccessKeysFragment/>
+
+## Using GitHub Actions
+
+Loft provides the following GitHub Actions for use in workflows:
+- [Setup DevSpace](https://github.com/loft-sh/setup-devspace): Installs the `devspace` CLI
+- [Setup Loft](https://github.com/loft-sh/setup-loft): Installs the `loft` CLI and logs in to the Loft instance
+- [Create Space](https://github.com/loft-sh/create-space): Creates a Loft Space
+- [Delete Space](https://github.com/loft-sh/delete-space): Deletes a Loft Space
+- [Create Virtual Cluster](https://github.com/loft-sh/create-vcluster): Creates a Loft Virtual Cluster
+- [Delete Virtual Cluster](https://github.com/loft-sh/delete-vcluster): Deletes a Loft Virtual Cluster
+
+### Using a Space for Pull Requests
+
+These examples show how to create and delete a Loft Space for pull requests.
+
+<Tabs
+    defaultValue="manual"
+    values={[
+        { label: 'Basic', value: 'manual', },
+        { label: 'Automatic Cleanup', value: 'automatic', },
+        { label: 'DevSpace', value: 'devspace', },
+    ]
+    }>
+<TabItem value="manual">
+
+This example shows how to create and delete a space to test an application named `my-app` for pull requests.
+
+```yaml
+# .github/workflows/prs.yaml
+name: Pull Request Checks
+on:
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Loft CLI
+        uses: loft-sh/setup-loft@main
+        with:
+          url: ${{ secrets.LOFT_URL }}
+          access-key: ${{ secrets.LOFT_ACCESS_KEY }}
+      - name: Create PR Space
+        uses: loft-sh/create-space@main
+        with:
+          name: pr-${{ github.event.pull_request.number }}-${{ github.sha }}-${{ github.run_id }}
+      - name: Deploy Application
+        run: kubectl apply -Rf ./kubernetes
+      - name: Wait for Deployment
+        run: kubectl rollout status -n pr-${{ github.event.pull_request.number }}-${{ github.sha }}-${{ github.run_id }} deployments/my-app
+      - name: Run Tests
+        run: make e2e
+      - name: Delete PR Space
+        uses: loft-sh/delete-space@main
+        with:
+          name: pr-${{ github.event.pull_request.number }}-${{ github.sha }}-${{ github.run_id }}
+```
+
+#### Step by Step Explanation
```

Comment:
No level-4 headlines here (also rest of the page)

Reply:
Convert to regular text (see devspace docs: https://devspace.sh/cli/docs/configuration/images/image-tagging#image-image-repository)